mailbox/opensearch/pom.xml

```diff
@@ -165,4 +165,16 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <reuseForks>true</reuseForks>
+                    <forkCount>2</forkCount>
```

Comment:
this is consensual ?

Reply:
I tried locally with `mvn surefire:test` (so no parallelism) , and adding forkCount = 2 would help me gain around 25% test time for a module. E.g 5 minute -> 3.5 minute. So I think increasing forkCount does help.

BTW, `forkCount` tuning seems classic in our project.